tests/unit_test.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 IBM, Red Hat
+# Copyright 2024 IBM, Red Hat
```

Comment:
Not sure about this update. Doesn't this reflect creation date? (Shouldn't matter really we are getting rid of this file anyway)

Reply:
Ohh so that's how it works, I reverted that now thanks!